### DISCUSSION TITLE
deprecated subdependencies

### DISCUSSION THREAD

**Discussion by haexhub** [mikro-orm/mikro-orm#5625] [Ideas]
### Describe the bug

Hi everyone,

in your project, you are using some deprecated sub/peer dependencies. It would be really nice, if you could bump them up.

_@mikro-orm/cli 6.2.8_
- glob 7.2.3
- rimraf 3.0.2
- inflight 1.0.6

_@mikro-orm/sqlite 6.2.8_
- @npmcli/move-file 1.1.2
- glob 7.2.3
- inflight 1.0.6
- rimraf 3.0.2

_@mikro-orm/knex_
- npmcli/move-file

### Reproduction

pnpm add @mikro-orm/core @mikro-orm/cli @mikro-orm/knex

### What driver are you using?

@mikro-orm/sqlite

### MikroORM version

6.2.8

### Node.js version

v21.2.0

### Operating system

Linux

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Slack](https://join.slack.com/t/mikroorm/shared_invite/enQtNTM1ODYzMzM4MDk3LWM4ZDExMjU5ZDhmNjA2MmM3MWMwZmExNjhhNDdiYTMwNWM0MGY5ZTE3ZjkyZTMzOWExNDgyYmMzNDE1NDI5NjA).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
those are not direct dependencies, i cant just bump them to a higher major version, that's not how it works. also all of those actually look like dev dependencies, why would you care about them?

**Reply by B4nan** [reply]
looks like most of those you listed are coming from jest and lerna, and we are already on latest versions of both of those packages

**Reply by haexhub** [reply]
I know that these are subdependencies. Just wanted to make you aware of it. Especially for inflight.

**Reply by B4nan** [reply]
> pnpm add @mikro-orm/core @mikro-orm/cli @mikro-orm/knex

This is actually not reproducing anything, you need to install the sqlite driver to see the warnings. And those are all caused by the sqlite dependency, again, we use latest version of it, nothing I can do about that. If you don't like the warning, there are two other sqlite drivers you can use.
